README.md

```diff
@@ -52,6 +52,21 @@ Add the following to `~/.boot/profile.boot`:
 (boot.core/load-data-readers!)
 ```
 
+### Shadow-CLJS
+
+Add the following to `shadow-cljs.edn`:
+```clojure
+{:dependencies [hashp "0.1.2"]
+ :nrepl {:init-ns user}
+ :builds {:app {:preloads [hashp.core]}}}
+```
+
+Create a file `src/user.clj`:
+```clojure
+(ns user
+  (:require [hashp.core]))
+```
+
```

Comment:
The setup for shadow-cljs is somewhat complex because we need to load both the clj code (for data reader) and the cljs code (for runtime values). 

I tried something like this but it doesn't work:

```clojure
;; hashp/core.cljs
(ns hashp.core
  (:require [zprint.core :as zprint])
  ;; There is no macros in hashp.core, we do this only to get this ns loaded into JVM.
  (:require-macros [hashp.core]))
```

I hoped the `:refer-macros` would load the `hashp.core` clj namespace, so the user would not need to load the `hashp.core` clj ns manually in their code, but it just didn't work.